app/views/admin/index.html.erb

```diff
@@ -0,0 +1,35 @@
+<h1>Requests</h1>
+
+<% if can? :manage, Request %>
```

Comment:
I think we should remove this authorization check and just count on the `authorize!` call in the index action.